frontend/package.json

```diff
@@ -22,9 +22,11 @@
   },
   "devDependencies": {
     "@nuxt/eslint": "^0.3.13",
+    "@nuxt/test-utils": "^3.14.1",
     "@nuxtjs/color-mode": "^3.3.2",
     "@nuxtjs/i18n": "^8.0.0",
     "@nuxtjs/style-resources": "^1.2.2",
+    "@playwright/test": "^1.47.0",
```

Comment:
dunno whats best, but just that you have seen it, could have also gone with `playwright-core` here 🤷